packages/jest-runtime/src/index.js

```diff
@@ -97,7 +97,9 @@ class Runtime {
   _mockFactories: {[key: string]: () => any, __proto__: null};
   _mockMetaDataCache: {[key: string]: MockFunctionMetadata, __proto__: null};
   _mockRegistry: {[key: string]: any, __proto__: null};
+  _sandboxMockRegistry: ?{[key: string]: any, __proto__: null};
   _moduleMocker: ModuleMocker;
+  _sandboxModuleRegistry: ?ModuleRegistry;
```

Comment:
I think `_isolatedModuleRegistry` would play slightly better with the name `isolateModules` but I'm good with both names 👍.

Reply:
👍 I like it way better!